libraries/AP_Scripting/lua_generated_bindings.cpp

```diff
@@ -13,7 +13,8 @@
 #include <AP_Common/Location.h>
 
 
-static int binding_argcheck(lua_State *L, int expected_arg_count) {
+static int binding_argcheck(lua_State *L, int expected_arg_count)
```

Comment:
You can't edit this file, you must edit the generator if you want to make a change.

Reply:
@WickedShell san.   thank you for your comment.
I adapted the generator to the ArduPilot coding style.